### ISSUE TITLE
why the test result of the point_pillar_v2vnet and point_pillar_intermediate_fusion all 0?

### ISSUE THREAD

**Issue by zllxot** [DerrickXuNu/OpenCOOD#37]
Hi, I trained the point_pillar_v2vnet model and evaluated it on the test set, but the evaluation results are all 0, the same is true for the point_pillar_intermediate_fusion model, what is the reason for this?
![image](https://user-images.githubusercontent.com/109151298/186156594-14776424-1c66-4ade-9cdd-9ff5d39d10fd.png)

**Comment by DerrickXuNu** [comment]
What spconv version did you use?

**Comment by zllxot** [comment]
> What spconv version did you use?

Thank you for your reply. My spconv version is 1.2.1.

**Comment by DerrickXuNu** [comment]
Can you run any results with our pretrained model? Also, is your training loss normal?

**Comment by zllxot** [comment]
> Can you run any results with our pretrained model? Also, is your training loss normal?

Yes, I get correct results when I test the pretrained model directly.
![image](https://user-images.githubusercontent.com/109151298/186309425-964506d7-a618-4009-99b4-66010fbd19d9.png)
The training loss also looks normal:
![image](https://user-images.githubusercontent.com/109151298/186310252-3210661c-8b29-46a0-9fc6-4812fe0e2432.png)

**Comment by DerrickXuNu** [comment]
How many epoches have you trained? What's your command to run the test?

**Comment by zllxot** [comment]
> How many epoches have you trained? What's your command to run the test?

Following the settings in the configuration file (point_pillar_v2vnet.yaml), I trained for a total of 60 epochs. 
The train command is:
`python opencood/tools/train.py --hypes_yaml /data1/zllxot/OpenCOOD/opencood/hypes_yaml/point_pillar_v2vnet.yaml`
The test command is:
`python opencood/tools/inference.py --model_dir /data1/zllxot/OpenCOOD/opencood/logs/point_pillar_v2vnet_2022_08_23_22_54_31/ --fusion_method intermediate`

**Comment by DerrickXuNu** [comment]
What is your loss in your last epoch?

**Comment by zllxot** [comment]
> What is your loss in your last epoch?

Sorry, I don't remember the loss in my last epoch, I'll train from scratch and get back to you.

**Comment by zllxot** [comment]
Hi, the training results seem to be normal this time. Thanks for your help.
